mattumland/next-graphcms-shows
# Show list view

## What issue does this solve?
https://codevalapp.github.io/next-graphcms-shows/issues/1/

## Background
Users need the option to view the upcoming shows in a list with additional details. 

## Manual testing instructions
1. Boot up the dev server
2. Go to `shows`
3. Click the `grid/list` toggle, confirm each view is working. 
4. Hover over list entries, confirm transitions are present
5. Click on list entries, confirm links are working
6. Confirm original grid view is unchanged.
